practitioner-service/src/main/resources/application-local.yml

```diff
@@ -46,7 +50,7 @@ azure:
     session-stateless: true
 practitioner:
   service:
-    uri: http://localhost:8081
+    uri: http://practitioner-service
```

Comment:
should this uri declaration be alongside the other uri declarations at the top of the file?

Reply:
Yes, fixed.